PerformanceTest_Static_Pressure_PlantarDorsal_ShinCalf_v2.py

```diff
@@ -0,0 +1,413 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jan 17 11:23:50 2023
+
+@author: Kate.Harrison
+"""
+
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import os
+import seaborn as sns
+from dataclasses import dataclass
+from tkinter import messagebox
+import scipy.signal as sig
+from XSENSORFunctions import readXSENSORFile, delimitTrial, createTSmat, zeroInsoleForce, findGaitEvents
+
+
+
+
+save_on = 1
+data_check = 1
+
+
+# Read in files
+# only read .asc files for this work
+
+fPath = 'C:\\Users\\bethany.kilpatrick\\BOA Technology Inc\\PFL Team - General\\Testing Segments\\Snow Performance\\2025_Performance_LassoPressure_Ride\\XSENSOR\\Exported\\'
+fileExt = r".csv"
+
+entries = [fName for fName in os.listdir(fPath) if fName.endswith(fileExt) and '0_' not in fName]
+entries_FrontBack = [fName for fName in os.listdir(fPath) if (fName.endswith(fileExt) and 'FrontBack' in fName)]
+entries_FootDorsum = [fName for fName in os.listdir(fPath) if (fName.endswith(fileExt) and 'FootDorsum' in fName)  ]
+
+
+### set plot font size ###
+SMALL_SIZE = 14
+MEDIUM_SIZE = 16
+BIGGER_SIZE = 18
+
+plt.rc('font', size=SMALL_SIZE)          # controls default text sizes
+plt.rc('axes', titlesize=SMALL_SIZE)     # fontsize of the axes title
+plt.rc('axes', labelsize=MEDIUM_SIZE)    # fontsize of the x and y labels
+plt.rc('xtick', labelsize=SMALL_SIZE)    # fontsize of the tick labels
+plt.rc('ytick', labelsize=SMALL_SIZE)    # fontsize of the tick labels
+plt.rc('legend', fontsize=SMALL_SIZE)    # legend fontsize
+plt.rc('figure', titlesize=BIGGER_SIZE)  # fontsize of the figure title
+
+@dataclass
+class FB_avgData:
+    
+    avgshinMat: np.array 
+
+    
+    avgcalfMat: np.array   
+
+    
+    config: str
+    subject: str
+    
+    # dat_FD: pd.DataFrame
```

Comment:
Would you mind removing commented out code that doesn't help with the interpretation of the code?

Reply:
ex: lines 61, 63-83, 94-99...